message/src/lib.rs

```diff
@@ -128,6 +134,6 @@ pub struct MessageHeader {
 /// As used by the `crate::v0` message format.
 #[derive(Debug, PartialEq, Eq, Clone)]
 pub struct AddressLookupTableAccount {
-    pub key: solana_pubkey::Pubkey,
-    pub addresses: Vec<solana_pubkey::Pubkey>,
+    pub key: Pubkey,
```

Comment:
now that the `Address` change has been split should this use that instead? I realize it's just an alias, but if we re-export I'd rather export the right one.